src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -184,6 +233,7 @@ def submit_to_testflight(
     def submit_to_app_store(
             self,
             build_id: ResourceId,
+            cancel_previous_submissions: bool = False,
             max_build_processing_wait: Optional[Union[int, Types.MaxBuildProcessingWait]] = None,
```

Comment:
For backwards compatibility it is better to retain the original method signature and ordering as much as possible. 
```suggestion
            max_build_processing_wait: Optional[Union[int, Types.MaxBuildProcessingWait]] = None,
            cancel_previous_submissions: bool = False,
```